docker/dockerfiles/Dockerfile.onnx.cpu

```
@@ -1,80 +1,70 @@
-FROM python:3.9 as base
-
+# Builder stage
+FROM python:3.13-slim AS builder
```

Inference is not yet compatible with python 3.13; lets use 3.12 instead

```suggestion
FROM python:3.12-slim AS builder
```